config.yml

```diff
@@ -49,3 +49,23 @@ scenario_params_list:
      - 10
    gradient_updates_per_pass_count:
      - 5
+ - dataset_name:
+    - 'imdb'
+   partners_count:
+    - 3
+   amounts_per_partner: 
+    - [0.2, 0.5, 0.3]
+   samples_split_option:
+    - ['basic', 'random']
+   multi_partner_learning_approach:
+    - 'fedavg'
+   aggregation_weighting:
+    - 'uniform'
+   methods:
+    - ["Shapley values", "Independent scores"]
+   epoch_count:
+    - 10
+   minibatch_count:
+    - 8
+   gradient_updates_per_pass_count:
+    - 5
```

Comment:
Adding `'imdb'` after line 32 would be a better factorization wouldn't it? 😃

Reply:
That seems to easy, It's probably a trap :rotating_light: